### ISSUE TITLE
Use Browser Location javascript error

### ISSUE THREAD

**Issue by JeroenED** [friendica/friendica#6511]
### Expected behavior
When enabling browser location in settings, you can write a new post, photo, etc while sharing exact location as well

### Actual behavior
When enabling "use browser location" the modal for writing the post doesn't appear

Debugger spits several Uncaught syntax error (Unexpected token &)

Sample of source:
````
if(navigator.geolocation) {
navigator.geolocation.getCurrentPosition(function(position) {
$(&#039;#jot-coord&amp;#039;).val(position.coords.latitude + &#039; &#039; + position.coords.longitude);
$(&#039;#profile-nolocation-wrapper&amp;#039;).show();
});
}
````

### Steps to reproduce the problem
* Enable "Use browser location" in settings (section basic)
* Try writing a post

### Friendica version you encountered the problem

2019.01 - Reported by Guy Van Sande (gvs@social.taurix.net) https://forum.friendi.ca/display/38381163-215c-472a-3eab-fcf099770352

2019.03-dev - Reproduced by @JeroenED

### Friendica source (git, zip)
2019.01 - Unknown
2019.03-dev - git

### PHP version
7.2 JeroenED

### SQL version
MariaDB-10.2

**Comment by MrPetovan** [comment]
Closed via #6512

**Comment by ToeiRei** [comment]
Still a bug on the current 'master' branch as of today
